### ISSUE TITLE
Interval fuzzing no longer works well

### ISSUE THREAD

**Issue by agroce** [crytic/echidna#460]
Given this contract:

```
pragma solidity ^0.4.0;

contract BlindAuction {
    uint public biddingEnd;

    bool bugged = false;

    modifier onlyBefore(uint _time) { if (now >= _time) throw; _; }

    function BlindAuction(
    ) {
	uint _biddingTime = 30000;
        biddingEnd = now + _biddingTime;
    }

    function bid()
        onlyBefore(biddingEnd)
    {
	bugged = true;
    }

    function echidna_bugged() public returns (bool) {
	return !bugged;
    }
}
```

echidna 1.4.0.0 release reliably finds a failing test, while latest master (probably after this:  https://github.com/crytic/echidna/pull/455/files) does not.

**Comment by ggrieco-tob** [comment]
I think I know this fails.

**Comment by ggrieco-tob** [comment]
It should be fixed in #469. Please test it.

**Comment by ggrieco-tob** [comment]
This should be fixed.
